pkg/store/bucket.go

```diff
@@ -237,6 +241,19 @@ func newBucketStoreMetrics(reg prometheus.Registerer) *bucketStoreMetrics {
 		Buckets: []float64{0.001, 0.01, 0.1, 0.3, 0.6, 1, 3, 6, 9, 20, 30, 60, 90, 120},
 	})
 
+	m.blockSyncs = promauto.With(reg).NewCounter(prometheus.CounterOpts{
+		Name: "thanos_bucket_store_block_syncs_successful_total",
+		Help: "Total number of successful block syncs",
```

Comment:
All the other help texts finishes with a `.`. Lets be consistent and add it here as well? Same for the other help texts.

Reply:
Updated.